internal-services/catalog/file-updates-pipeline.yaml

```diff
@@ -1,57 +0,0 @@
----
-apiVersion: tekton.dev/v1beta1
-kind: Pipeline
-metadata:
-  name: file-updates
```

Comment:
I think we need to change the migrated version to this name before we remove this here

Reply:
done here: https://github.com/konflux-ci/release-service-catalog/pull/682/files#diff-50b7aa963fdf55a7ea15a3ad71d8cd3b3b8d314d07fea08bf2acbbba9ef3bc68R28